hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SimplePipelineProvider.java

```diff
@@ -72,4 +72,9 @@ public Pipeline create(ReplicationFactor factor,
         .setNodes(nodes)
         .build();
   }
+
+  @Override
+  public void shutdown() {
+
```

Comment:
NIT - Can we add a comment like //do nothing.

Reply:
Done